README.md

```diff
@@ -305,6 +305,30 @@ to your local platform and then build (Adopt) OpenJDK and place the result in
 These scripts default to using AdoptOpenJDK as the OpenJDK source repo to build 
 from, but you can override this with the `-r` flag.
 
+#Metadata
+Along side the built assets a metadata file will be created with info about the build. This will look of the form:
```

Comment:
* "Along side" is one word I think
* Suggestion: _"This will be a JSON document of the form"_